### PR TITLE
update mir version

### PR DESCRIPTION
**Pull request by 9il**
@ShigekiKarita we are moving part of awesome numir API and examples to mir-algorithm.

### CODE REVIEW

File: source/numir/stats.d [hunk 1]
```diff
@@ -619,6 +614,7 @@ nothrow pure @nogc template center(sumTemplateArgs...)
     +/
     auto center(S, Seed)(S slice, Seed seed) if (isSlice!S)
```
Comment (jmh530) [inline source/numir/stats.d:615]: This needs to be removed. mir.stat.mean removed seed.

File: source/numir/stats.d [hunk 2]
```diff
@@ -685,6 +682,7 @@ pure nothrow @nogc private template deviationsPow(sumTemplateArgs...)
     auto deviationsPow(S, Order, Seed)
```
Comment (jmh530) [inline source/numir/stats.d:682]: Same as above.

File: source/numir/stats.d [hunk 3]
```diff
@@ -759,6 +758,7 @@ pure nothrow @nogc template moment(sumTemplateArgs...)
     auto moment(S, Order, Seed)
```
Comment (jmh530) [inline source/numir/stats.d:758]: Same as above.

File: source/numir/stats.d (hunk 1 above)
Comment (9il) [reply source/numir/stats.d:615]: done in #40

File: source/numir/stats.d (hunk 2 above)
Comment (9il) [reply source/numir/stats.d:682]: done in #40

File: source/numir/stats.d (hunk 3 above)
Comment (9il) [reply source/numir/stats.d:758]: done in #40

**Pull request merged**